Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -374,6 +374,33 @@ In Fluid, you can now use the defined language key ("language"):
 
     <f:translate languageKey="{language}" id="LLL:EXT:my_ext/Resources/Private/Language/emails.xml:subject" />
 
+..  _mail-fluid-email-set-request:
+
+Set the current request object for `FluidEmail`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to use ViewHelpers that need a valid current request, such as :ref:`t3viewhelper:typo3-fluid-uri-page`,
+pass the current request to the FluidEmail instance:
+
+
+..  code-block:: php
+
+    use TYPO3\CMS\Core\Mail\FluidEmail;
+
+    $email = new FluidEmail();
+    $email->setRequest($this->request);
+
+Read more aboout :ref:`Getting the PSR-7 request object <getting-typo3-request-object>` in different 
+contexts. In a context where no valid request object can be retrieved, such as in a 
+:ref:`Console command <t3coreapi:symfony-console-commands>` the affected ViewHelpers cannot be used.
```

Comment:
Are you sure? You can construct a request object manually. What about the password forgot command? Does it not use FluidEmail?

Reply:
You can send FluidEmails without request object but not use certain viewhelpers then.